src/rest-server/docs/swagger.yaml

```diff
@@ -419,14 +419,16 @@ paths:
         - virtual cluster
       summary: Get the list of virtual clusters.
       description: Get the list of virtual clusters.
-      operationId: getVirtualClusters
+      operationId: v2getVirtualClusters
```

Comment:
In other opeations we skip `v2` because this file is for v2 only. Let's keep insistence.

Reply:
updated